doc/book/src/development/operator.md

```diff
@@ -0,0 +1,23 @@
+# Operator
+
+The [operator code][operator-folder] is using the [Operator-SDK][operator-sdk].
+
+* the [`api` folder][operator-api-folder] contains the Go code for Operator's CRDs
+* the [`config` folder][operator-config-folder] contains the YAML manifests
+* the [`internal` folder][operator-internal-folder] contains the code for the reconcilers 
+
+## Run Tests
+
+To run Unit tests you can execute the `make test` command.
```

Comment:
```suggestion
To run unit tests you can execute the `make test` command.
```

Also, it's at this point that I realize we should probably have a top-level `unit-test` rule that runs all unit tests.  IMO there's not any real good reason that unit tests shouldn't run everything.

Reply:
typo fixed in #224.

I agree on the top level `unit-test` rule. I'll open a PR for this.

Reply:
PR for top-level `unit-test` rule: #225